specs/0108-community-health-dashboard.md

```diff
@@ -0,0 +1,49 @@
+- Feature Name: Community Health Dashboard
+- Created: 2020-10-23
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1672966
+
+# Summary
+
+Create a team-specific health dashboard targeted at Pontoon users with Manager rights.
+
+# Motivation
+
+Team managers as well as project managers should have the ability to understand how localization communities are performing. Team dashboards already provide some of the answers, including the overall status of completion and translation statistics, but the information is only available in the form of a current snapshot.
```

Comment:
This is a little bit of a nit, but could we reframe this as "engagement" instead of "performance"? I know that I've historically used that term, but I've realized that there's a nuance to the term "performance" that relates to an employer - employee relationship. We should seek to avoid that and speak of this in terms of how much the community is engaged in our localization initiatives.